GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -119,6 +119,9 @@ public class Glia {
 
     private(set) var configuration: Configuration?
 
+    // Interface for entry widget
+    public var entryWidget: EntryWidget
```

Comment:
Could we make it `public let` or `public private(set) var`?

Reply:
fixed [d6cdd68](https://github.com/salemove/ios-sdk-widgets/pull/1050/commits/d6cdd68c41e5fea8e071da56a345aa320bab520f)